tests/test_basic.py

```diff
@@ -29,8 +30,8 @@ def test_update_normal():
     assert round(corr_list["xx1"][1], 1) == -0.5
     assert round(corr_list["xx1"][2], 1) == 0.3
 
-## Test handling of variable lists
-def test_predictor_normal_handling():
+## Test handling of variable lists              ###??? What does it mean by 'variable lists'?
+def test_predictor_normal_handling():           ###??? Why don't we put all the update_predictor_normal tests in one function?
```

Comment:
Reply to _"Why don't we put all the update_predictor_normal tests in one function?"_:
It's just personal coding preference. It would be much easier and clear for me to classify unit test.

Reply:
Got it, thanks for the clarification!